MH4U Dex/RegionEncylcopediaCollectionViewCell.m

```diff
@@ -8,6 +8,26 @@
 
 #import "RegionEncylcopediaCollectionViewCell.h"
 
+@interface RegionEncylcopediaCollectionViewCell ()
+
+@property (nonatomic, weak) IBOutlet UILabel *regionNameLabel;
+@property (nonatomic, weak) IBOutlet UIImageView *regionIconImageView;
+
+@end
+
 @implementation RegionEncylcopediaCollectionViewCell
 
+- (void)displayContents
+{
+    NSString *imageName = [NSString stringWithFormat:@"%@-icon", self.regionKeyName];
```

Comment:
```objc
NSString *imageName = [self.regionKeyName stringByAppendingString:@"-icon"];
```
Also, you've got basically this same construct in two far-apart places, so it might make sense to consolidate this logic in one more-centralized place.